styles/_theme.scss

```diff
@@ -125,3 +125,19 @@ $excludedLinks: ".btn, .iconBtn, .listItem";
 .clrSh2  {
```

Comment:
One suggestion I wanted to make for customizing the theme would be to use template to generate the styles, for example:

```
// theme template
.clrP {
  background-color: <%= ob.clrPrimary %>;

  input[type="radio"] + label:before {
    background: <%= ob.clrSecondary %>;
  }

  input[type="radio"]:checked + label:before {
    box-shadow: inset 0 0 0 3px <%= ob.clrSecondary %>;
  }
}

...
```

```
// when a theme change is needed
$('#themeStyle').remove(); 
$('<style id="themeStyle" />').text(themeTmpl.render(profile.toJSON()))
  .appendTo($('head')); // we should cache the $('head') selector

// I'm assuming the theme color would come from some user's profile model, but we can adjust if this is not the case.